pkg/driver/validation_test.go

```diff
@@ -66,6 +66,13 @@ func TestValidateExtraVolumeTags(t *testing.T) {
 			},
 			expErr: fmt.Errorf("Tag key too long (actual: %d, limit: %d)", cloud.MaxTagKeyLength+1, cloud.MaxTagKeyLength),
 		},
+		{
+			name: "invaid tag: key is empty",
+			tags: map[string]string{
+				randomString(cloud.MinTagKeyLength - 1): "extra-tag-value",
```

Comment:
this becomes `randomString(0)` which is always just `""`

Can we instead do `"": "extra-tag-value",` to make it obvious to readers what this test is doing?